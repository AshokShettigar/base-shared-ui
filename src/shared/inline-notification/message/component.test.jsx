import { shallow } from 'enzyme';
import InlineNotificationMessage from './component';


const defaultProps = {
    children: 'Inline Notification Message'
};

describe('Inline Notification Message', () => {
    it('default renders', () => {
        const wrapper = shallow(<InlineNotificationMessage {...defaultProps} />);
        expect(wrapper).toMatchSnapshot();
    });

    it('renders with prop className', () => {
        const customProps = {
            className: 'class-name'
        };

        const wrapper = shallow(<InlineNotificationMessage {...defaultProps} {...customProps} />);

        expect(wrapper).toMatchSnapshot();
    });
});

