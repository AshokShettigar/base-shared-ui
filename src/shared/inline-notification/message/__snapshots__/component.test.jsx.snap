// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Inline Notification Message default renders 1`] = `
<div
  className="inline-notification__message"
  data-component="InlineNotificationMessage"
>
  Inline Notification Message
</div>
`;

exports[`Inline Notification Message renders with prop className 1`] = `
<div
  className="inline-notification__message class-name"
  data-component="InlineNotificationMessage"
>
  Inline Notification Message
</div>
`;
